### DISCUSSION TITLE
Is there a way to access system clipboard?

### DISCUSSION THREAD

**Discussion by dxtr85** [bevyengine/bevy#7074] [Q&A] [answered]
Some users of this game engine might find being able to paste from or copy into system clipboard very useful. 
Especially in case someone is working on an app that is a tool and not necessarily a game.
Does bevy currently have support for system clipboard?
If not is it planned to be introduced?
Is there some other way to access system clipboard that is platform agnostic?

**Comment by frammiie** [comment, marked as answer]
This isn't really related to bevy, would recommend a crate such as:

https://crates.io/crates/clipboard-win
or
https://crates.io/crates/clipboard

depending on your needs/usecase.

**Reply by dxtr85** [reply]
Thanks!
